### PR TITLE
Load bg from tmx

### PR DESCRIPTION
**Pull request by Hattes**

### CODE REVIEW

**Review by feha — APPROVED** [review]

File: PixelHunter1995/SceneLib/SceneHandler.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    class SceneHandler
+    {
+        //List<Scene> scenes;
+        public Dictionary<string, Scene> scenes;
+        public Scene currentScene;
+
+        public SceneHandler()
+        {
+        }
+        public void Initialize(string scenesPath)
```
Comment (Torakko) [inline PixelHunter1995/SceneLib/SceneHandler.cs]: missing empty line

File: PixelHunter1995/SceneLib/SceneHandler.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace PixelHunter1995
+{
+    class SceneHandler
+    {
+        //List<Scene> scenes;
```
Comment (Torakko) [inline PixelHunter1995/SceneLib/SceneHandler.cs]: remove?

File: PixelHunter1995/SceneLib/SceneHandler.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
```
Comment (Torakko) [inline PixelHunter1995/SceneLib/SceneHandler.cs]: rename to SceneManager

File: PixelHunter1995/SceneLib/Scene.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+using System.Net.Mime;
+
+namespace PixelHunter1995
+{
+    using Dog = System.ValueTuple<float, float, float, float>;
+
+    //using Background = System.ValueTuple<>;
```
Comment (Torakko) [inline PixelHunter1995/SceneLib/Scene.cs]: remove?

File: PixelHunter1995/Main.cs [hunk 5]
```diff
@@ -56,15 +56,18 @@ protected override void LoadContent()
             spriteBatch = new SpriteBatch(graphics.GraphicsDevice);
 
             // Load images
-            Texture2D background = Content.Load<Texture2D>("Images/Background1");
             Texture2D menu = Content.Load<Texture2D>("Images/Menu");
             Texture2D guy = Content.Load<Texture2D>("Images/snubbe");
 
             // Load sounds
             music = Content.Load<SoundEffect>("Sounds/slow-music");
```
Comment (Torakko) [inline PixelHunter1995/Main.cs:63]: add empty line

**Review by Torakko — APPROVED** [review]

**Pull request merged**